build/php.m4

```diff
@@ -1698,7 +1698,7 @@ AC_DEFUN([PHP_C_BIGENDIAN],
  [
   ac_cv_c_bigendian_php=unknown
   AC_RUN_IFELSE([AC_LANG_SOURCE([[
-int main(void)
+int main(int argc, char** argv)
```

Comment:
Isn't ``int main(void)`` one of the valid signatures of the main function?

Reply:
Yes it looks like you're right in the C23 draft. I'm happy to leave that line alone, or even to make them all `int main(void)`.

Reply:
I think changing them all to ``main(void)`` would be my preference. Also, I'm pretty sure this is part of the specification since forever (and definitely since C99). But good to know C23 doesn't change this!

Reply:
I worded that rather badly. I just meant that the freely available C23 draft was what was easiest for me to check.

I've updated them all to `int main(void)`.